.gitignore

```diff
@@ -1,9 +1,12 @@
+
+# Secret
+.env
+
 # Build Artifacts
 /target
 
 # Luro Specific
-!/data
-/data/*
+/data
```

Comment:
Empty folders arent pushed. So having a carveout for data didnt do anything. Could add an empty file in /data or do some git wizardry, but now the code creates /data if it doesn't exist.

Reply:
I'm aware empty folders are not pushed, in mainline Luro there is the sample folder however. This pretty much matched my setup there, and once I add `/data/sample` things will be correct. That raises another question though... Maybe it would be better to force include `!/data/sample` and exclude `/data`?

Thoughts?

Reply:
Either would work, But I think just !/data/sample once/if we make it.
Then no custom secrets would be made visible by accident.

Reply:
Sounds good to me!